src/bitgo.js

```diff
@@ -173,7 +173,8 @@ const BitGo = function(params) {
     params.customBitcoinNetwork ||
     params.customSigningAddress ||
     process.env.BITGO_CUSTOM_ROOT_URI ||
-    process.env.BITGO_CUSTOM_BITCOIN_NETWORK) {
+    process.env.BITGO_CUSTOM_BITCOIN_NETWORK ||
+    params.microservicesUri) {
```

Comment:
How come passing in a microservicesUri forces me into the custom env?

Reply:
my thought was because if you're passing in a microservices uri you must be in a custom environment. if you're not in a custom environment, then you don't have a need to talk to the microservices.